docs/tables/oci_cloud_guard_detector_recipe.md

```diff
@@ -0,0 +1,31 @@
+# Table: oci_cloud_guard_detector_recipe
+
+OCI Cloud Guard detector recipe uses multiple detector rules, each of which a specific definition of a class of resources, with specific actions or configurations, that cause a detector to report a problem. If any one rule is triggered, it causes the detector to report a problem.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  time_created,
+  lifecycle_state as state
+from
+  oci_cloud_guard_detector_recipe;
+```
+
+### List detector recipe which are active
+
+```sql
+select
+  name,
+  id,
+  time_created,
+  lifecycle_state as state
+from
+  oci_cloud_guard_detector_recipe
+where
+  lifecycle_state = 'ACTIVE';
+```
```

Comment:
Add additional example to filter the  rule name called `Password policy does not meet complexity requirements` OR    `Password is too old`

Also if we can print the values of the rule set i.e. 
![Screenshot 2021-04-22 at 3 32 31 PM](https://user-images.githubusercontent.com/26873346/115696148-029e4400-a380-11eb-8ad2-28781a4778fe.png)

Reply:
as discussed , updated .